src/panels/config/application_credentials/dialog-add-application-credential.ts

```diff
@@ -61,11 +69,15 @@ export class DialogAddApplicationCredential extends LitElement {
   }
 
   private async _fetchConfig() {
-    const config = await fetchApplicationCredentialsConfig(this.hass);
-    this._domains = config.domains.map((domain) => ({
-      id: domain,
-      name: domainToName(this.hass.localize, domain),
-    }));
+    this._config = await fetchApplicationCredentialsConfig(this.hass);
+    this._domains = Object.entries(this._config.integrations).map(
+      ([domain, info]) => ({
+        id: domain,
+        name: domainToName(this.hass.localize, domain),
+        description_placeholders: info.description_placeholders,
```

Comment:
This is never used? We always use `this._config` directly when fetching `description_placeholders`?

Reply:
Yes, good catch, this was my first attempt.  Removed!